frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -52,10 +56,11 @@ const Studio: React.FC = () => {
                     />
                 }
                 right={
-                    <IconButton
+                    <Button
                         variant="primaryOutlined"
-                        aria-label="Download"
-                        icon={<Icon as={IconName.DOWNLOAD} />}
+                        label="Submit"
+                        sx={{ width: '100px' }}
```

Comment:
we dont use sx props!

Reply:
I think it would be better to fix it once [178](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/178) is merged as a merge conflict because if I change it here Ill have to change the button component and all components that use the button..